scripts/generate-static-data.js

```diff
@@ -150,6 +150,8 @@ const texts = [
     'High-Close',
     'Close-Low',
     'High-Low',
+    '"AUD/JPY" or "Apple"',
```

Comment:
where is this string being used?

Reply:
@4p00rv , it's used as the placeholder search string for asset dropdown right?